spring-cloud-gcp-starters/spring-cloud-gcp-starter-config/src/test/java/org/springframework/cloud/gcp/config/GoogleConfigPropertySourceLocatorTest.java

```diff
@@ -84,6 +84,15 @@ public void disabledPropertySourceReturnsNull() throws Exception {
 		verify(this.googleConfigPropertySourceLocator, never()).getRemoteEnvironment();
 	}
 
+	@Test
+	public void disabledPropertySourceAvoidChecks() throws Exception {
+		this.gcpConfigProperties.setEnabled(false);
```

Comment:
Why not use `loadEnvironment("spring.cloud.gcp.config.enabled=false")` instead?

Reply:
This is different test class and there is no such a method yet.
I think this is kinda unit test, so that's fine.
However I agree that `GcpConfigAutoConfigurationTest` should have one more test about the fully blown Spring Boot environment for the `spring.cloud.gcp.config.enabled=false` case.

Reply:
I'm not too sure what you mean with that additional test case.
Can you explain?

Reply:
OK. This test class is about unit-tests, just for the `GoogleConfigPropertySourceLocator` functionality verifications. There is no `ApplicationContext` involved, so @meltsufin 's `loadEnvironment()` cannon be here. 
We can add a requested test-case, fully blown Boot test, into the `GcpConfigAutoConfigurationTest`. There is one for the `spring.cloud.gcp.config.enabled=true`, but there is not already one for the `spring.cloud.gcp.config.enabled=false`.
That's what I mean. Does it make sense now?